### DISCUSSION TITLE
Range Node

### DISCUSSION THREAD

**Discussion by mtoko** [phoddie/node-red-mcu#7] [General]
I followed Peter's advice and was able to add the code for (Scale and Limit)  and (Scale and Wrap) on the Range Node. Any other nodes I could try? They would need to be reasonably simple or adding features to and existing node.

**Comment by phoddie** [comment]
> ... was able to add the code for (Scale and Limit) and (Scale and Wrap) on the Range Node

Congratulations on adding the additional options to the Range Node! Would you be willing to open a Pull Request so that I can merge the changes to the repository for everyone to use?

FYI – please also complete Moddable's [Contributor License Agreement](https://github.com/Moddable-OpenSource/moddable/tree/public/licenses#contributor-license-agreement) (CLA). THat's required for us to accept PRs.

> Any other nodes I could try? They would need to be reasonably simple or adding features to and existing node.

The JSON node might be an interesting node to try to implement. It looks to be reasonably straightforward. The schema feature is a big external dependency though, and that should probably be skipped for now.

**Reply by mtoko** [reply]
I have done the pull request, please check carefully, I am learning as I go. I will look at the CLA and send back to you.
